docs/en/integrations/language-clients/nodejs.md

```diff
@@ -7,82 +7,113 @@ description: The official Node.js client for connecting to ClickHouse.
 ---
 
 # ClickHouse JS
+
 The official Node.js client for connecting to ClickHouse.
 The client is written in TypeScript and provides typings for the client public API.
 
 ## Environment requirements
-The nodejs must be available in the environment to run the client.
-The client is compatible with all the [maintained](https://github.com/nodejs/release#readme) nodejs LTS versions: `v14.x` and `v16.x`.
 
-As soon as a nodejs version approaches End-Of-Life, the client drops support for the outdated insecure nodejs version.
+The Node.js must be available in the environment to run the client.
+The client is compatible with all the [maintained](https://github.com/nodejs/release#readme) Node.js LTS
+versions: `v14.x` and `v16.x`, as well as the current `v18.x`.
```

Comment:
```suggestion
versions.
```

Reply:
up to you, but you could just say LTS versions and not have to worry about updating the doc when 14 drop s off.

Reply:
I think we should just remove it indeed.